api/src/main/java/gov/va/vro/api/resources/VroResource.java

```diff
@@ -68,7 +68,6 @@ ResponseEntity<GeneratePdfResponse> generatePdf(
       description =
           "Downloads the evidence pdf previously generated by the posted patient health data and diagnostic code")
   @GetMapping("/evidence-pdf/{claimSubmissionId}")
-  @ResponseStatus(HttpStatus.OK)
```

Comment:
This is only for Swagger documentation. I think we might want to find out how to list multiple responses instead of removing the only one we have.

Reply:
You are right. ResponseStatus is not what I thought it was. It is better to remove it I agree.